public/docs/ts/latest/guide/server-communication.jade

```diff
@@ -19,37 +19,46 @@ block includes
   * [Demos](#demos)
   * [Providing HTTP Services](#http-providers)
   * [The Tour of Heroes *HTTP* client demo](#http-client)
-    - [The `HeroListComponent` class](#HeroListComponent)
+
+    * [The `HeroListComponent` class](#HeroListComponent)
+
   * [Fetch data with `http.get()`](#fetch-data)
   <li>[RxJS library](#rxjs-library)
     <ul>
       <li> [Enable RxJS operators](#enable-rxjs-operators)</li>
     </ul>
   </li>
+
   * [Process the response object](#extract-data)
-    - [Parse to `JSON`](#parse-to-json)
-    - [Do not return the response object](#no-return-response-object)
-    - [Always handle errors](#error-handling)
-    - [`HeroListComponent` error handling](#hero-list-component)
+
+    * [Parse to `JSON`](#parse-to-json)
+    * [Do not return the response object](#no-return-response-object)
+    * [Always handle errors](#error-handling)
+    * [`HeroListComponent` error handling](#hero-list-component)
+
   * [Send data to the server](#update)
-    - [Headers](#headers)
-    - [JSON results](#json-results)
+
+    * [Headers](#headers)
+    * [JSON results](#json-results)
 
   <ul><li> [Fall back to promises](#promises)</ul>
 
   * [Cross-Origin Requests: Wikipedia example](#cors)
+
   <ul>
     <li> [Search Wikipedia](#search-wikipedia)</li>
     <li> [Search parameters](#search-parameters)</li>
     <li> [The WikiComponent](#wikicomponent)</li>
   </ul>
+
   * [A wasteful app](#wasteful-app)
   <li> [More fun with Observables](#more-observables)
       <ul>
         <li> [Create a stream of search terms](#create-stream)</li>
         <li> [Listen for search terms](#listen-for-search-terms)</li>
       </ul>
   </li>
```

Comment:
Markdown sucks with lists. Well known. I guess I gave up trying to make it work and switched.
We'll have to do that in `aio` quite a bit more I fear.